src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
```

Comment:
Ensure that the `PostList` component is correctly implemented in the `./components/PostList` file.